packages/backend/src/controllers/authentication.ts

```diff
@@ -164,18 +168,27 @@ export const googlePassportStrategy: GoogleStrategy | undefined = !(
               ).href,
               passReqToCallback: true,
           },
-          async (req, issuer, profile, done) => {
+          async (
+              req: Express.Request,
+              accessToken: string,
+              refreshToken: string,
+              params: GoogleCallbackParameters,
+              profile: Profile,
+              done: VerifyCallback,
+          ) => {
               try {
+                  const issuer = 'https://accounts.google.com';
```

Comment:
✅ Verified this is also hardcoded in the `passport-google-oidc` package, so this works